util/metrics/src/service_metrics.rs

```diff
@@ -100,17 +119,25 @@ impl ServiceMetrics {
         )
     }
 
+    /// Count number of errors by method
```

Comment:
I'm guessing this method is called when a gRPC response is received, and isn't only called to count number of errors by method? If so, maybe add a sentence before this sentence to ensure that the reader understands the full context.

Reply:
I could expand on the comment